pengwin-setup.d/common.sh

```diff
@@ -180,7 +180,7 @@ function menu() {
 function setup_env() {
 
   if (! command -v cmd.exe >/dev/null); then
-    whiptail --title "Wrong user" --msgbox "pengwin-setup was ran with the user \"${USER}\".\n\nRun pengwin-setup from the default user and without sudo" 12 80
+    whiptail --title "An environment problem was found" --msgbox "The Windows PATH is not available, and pengwin-setup requires it to run. Please check that: \n\n   pengwin-setup is not running with sudo.\n   pengwin-setup wasn't run with the root user.\n   appendWindowsPath is true in /etc/wsl.conf file or is not defined. \n\n\nIf you don't want to have Windows PATH in Pengwin, enable it temporally to run pengwin-setup" 15 100
```

Comment:
`temporally` 

should be temporarily.

Reply:
Thank you we will correct this

Reply:
And this is, of course, while we take the time to make pengwin-setup work without Windows path using the techniques that wslu make.